_posts/2020-09-03-tutorial-syllabics-fonts.md

```diff
@@ -0,0 +1,283 @@
+<link rel="stylesheet" href="//fonts.googleapis.com/earlyaccess/notosanscanadianaboriginal.css">
+```
+
+Read the rest of this article to learn more about choosing syllabics
+fonts for the web!
+
+## What you need to know to understand this post
+
+You will need to know how to:
+
+ * How to edit your website's <abbr title="Cascading Stylesheets">[CSS][]</abbr>
+ * How to add a [`<link>`][link] element to your HTML's `<head>`
+
+If you've ever added a custom font from [Google Fonts][] to your
+website, you know enough to follow this post!
+
+It's also helpful to know the [difference between serif and sans-serif
+fonts][serif-vs-sans-serif].
+
+[CSS]: https://www.w3schools.com/css/css_howto.asp#:~:text=With%20an%20external%20style%20sheet,%20you%20can%20change%20the%20look%20of%20an%20entire%20website%20by%20changing%20just%20one%20file!
+[link]: https://www.w3schools.com/tags/tag_link.asp
+[Google Fonts]: https://fonts.google.com/
+[serif-vs-sans-serif]: https://about.easil.com/support/serif-vs-sans-serif/
+
+## Let's get started!
+
+Have you ever wanted to display **syllabics** on your website, but
+when you tried, all you got was [tofu]:
+
+> □□□ □□□□□□! □□□□□□□□□□ □□□□□ «□ □□□□□□□□□»□
+
+[tofu]: https://english.stackexchange.com/a/331483
+
+Tofu no longer! Let's discuss the various syllabics fonts that exist out
+there, where they are commonly found, and let's create a **font stack**:
+a list of fonts that your web browser tries in order such that it finds
+a font that displays syllabics correctly.
+
+## Common syllabics fonts
+
+I am aware of the following fonts:
+
+ - **Aboriginal Sans** and **Aboriginal Serif**
+ - **Euphemia**
+ - **Gadugi**
+ - **Noto Sans Canadian Aboriginal**
+
+I made an **interactive demonstration** of these fonts that you can play
+with. Go ahead and type any text in (western) Cree into the box and it
+will be converted to syllabics and displayed in the appropriate font, if
+it's available on your system:
+
+<p class="codepen" data-height="500" data-theme-id="light" data-default-tab="result" data-user="eddieantonio" data-slug-hash="GRZOBYY" style="height: 500px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;" data-pen-title="Syllabics font playground">
+  <span>See the Pen <a href="https://codepen.io/eddieantonio/pen/GRZOBYY">
+  Syllabics font playground</a> by Eddie Antonio Santos (<a href="https://codepen.io/eddieantonio">@eddieantonio</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
+</p>
+
+---
+
+But **where** do these fonts come from? Where should you **expect these fonts to be
+available**?
+
+
+### Aboriginal Sans and Aboriginal Serif
+
+As the names suggest, they have both serif and sans-serif variants for
+writing in Latin script (e.g., English, French, etc.).
+
+#### Who created these fonts?
+
+These fonts were created by the Language Geek himself, [Chris Harvey][].
+
+#### Where can I find these fonts?
```

Comment:
I think "Where can I find these fonts?" threw me a bit when I first read it, because I was expecting all of the answers to refer to places that you could download fonts. Would "Where can I expect to find these fonts?" or "Where can I expect to find these fonts pre-installed?" or even "Which users are likely to have these fonts pre-installed?" work? I think it would set expectations more clearly. (Also, I have to admit that I had really not spent that long thinking about fonts on websites before reading this, and since where I'm coming from is thinking about using fonts in compiled LaTeX, I'm often thinking about _manually_ adding the fonts to a document that I will compile and for which the fonts will be embedded in the document, rather than thinking about how a website will be displayed to a user whose selection of fonts I don't control. I wonder if you might want to add to "things you need to know" an understanding of how fonts are displayed on webpages? Maybe a knowledge of CSS already includes this, or maybe you could link something like: https://en.wikipedia.org/wiki/Web_typography or a better alternative.)

Reply:
Haha, I'm think about minimum effort for both the site maintainer and visitors to your site; so I'm shying away from manually downloading and installing fonts in this blog post. I know I've made work for myself to create a _review_ of different syllabics fonts, including the ones you can download (à la https://www.eastcree.org/cree/en/resources/how-to/cree-fonts/#Afonts but updated for 2020).

Reply:
To clarify, that was mostly to provide context about why I was slightly confused about this wording; I don't think you need to add that to this post!

Reply:
Oh, but I like this wording better! 😄